gate_radius: 0.5
use_lane_detection: false